tests/test_queries.py

```diff
@@ -1158,31 +1235,103 @@ def test_words(client: FlaskClient):
     assert json["voice"].startswith("Nafnorðið „splorglobb“ fannst ekki")
 
 
-def test_yulelads(client: FlaskClient):
-    """ Yule lads module """
+def test_yulelads(client: FlaskClient) -> None:
+    """Yule lads module"""
+
     qmcall(
         client,
         {"q": "hvenær kemur fyrsti jólasveinninn til byggða", "voice": True},
         "YuleLads",
     )
 
 
-def _cleanup():
-    """Delete any queries or query data logged as
-    result of query module tests"""
-    with SessionContext(commit=True) as session:
-        session.execute(
-            Query.table().delete().where(Query.client_id == DUMMY_CLIENT_ID)
-        )
-        session.execute(
-            QueryData.table().delete().where(QueryData.client_id == DUMMY_CLIENT_ID)
-        )
-        # Note: there is no client_id in the querylog table
-        # so we cannot delete the logged queries there by this criterion.
+# NB: Do not move this function. Pytest runs tests in the order they
+# appear in the source file, and this test function should be the
+# last to run, since it has the fortuitous side effect of deleting
+# any logged queries/query data saved to the database due to tests.
```

Comment:
Is there some kind of Pytest teardown function that could be defined to do this, i.e. delete query data from tests? Sometimes we want to run individual tests, not the whole suite, and such an automatic teardown would be nice in those cases.

Reply:
That's a good point which I did give some thought to at the time. Will put on my TODO list.